comit/src/network/protocols/orderbook.rs

```diff
@@ -1,5 +1,5 @@
 mod order;
-mod take_order;
+pub mod take_order;
```

Comment:
Why do you guys need access to take order? Afaik the functionality is exposed through the orderbook module?